aws-java-sdk-core/pom.xml

```diff
@@ -21,7 +21,12 @@
     <dependency>
       <groupId>org.apache.httpcomponents</groupId>
       <artifactId>httpclient</artifactId>
-      <version>4.3.6</version>
+      <version>${httpclient.version}</version>
```

Comment:
I was thinking maybe:

httpcomponents.httpclient.version

https://hc.apache.org/

To disambiguate from the older Apache httpclient.